doc/setup_instructions.md

```diff
@@ -486,3 +485,117 @@ build tool option, if it isn't already.
 
 Your locally cloned Thrive version may get messed up from time to time.
 [Here are the steps to fix it.](https://wiki.revolutionarygamesstudio.com/wiki/Cleaning_Local_Thrive_Version)
+
+Working with translation
+========================
+
+Thrive is using gettext as a localization tool. gettext works by having a
+template file (of extension type .pot) and then one translation file for
+each language (of extension type .po)
+
+Required tools
+--------------
+
+If you are planning to do anything that would require translation, or
+simply to translate the game into your locale language you may need
+a few more tools.
+
+Note if you are simply planning to add a new localization, Poedit is
+enough.
+
+### Python
+
+The tool used to extract strings from the game files is using
+[Python](https://www.python.org/downloads).
+So if you are planning to add or edit strings into the game, you'll need it.
```

Comment:
Python 3?

Reply:
Also add a note that on Linux you probably have this installed already, if not use package manager to get `python3` package.